### ISSUE TITLE
email and password env not working properly after docker install

### ISSUE THREAD

**Issue by jaimezapataspg** [chiefonboarding/ChiefOnboarding#254]
## Describe the bug
Cheifonboarding is not responding to the `ACCOUNT_EMAIL` and `ACCOUNT_PASSWORD` setup on the env file.

## To Reproduce
Install via Docker Compose using "[2.2 Non-caddy-based deployments](https://docs.chiefonboarding.com/deployment/#22-non-caddy-based-deployment)" template from the documentation with Portainer, including env variables in the docker-compose file.

## Expected behavior
Access via the app via internal IP and use the configured env to log in

## Current behavior
A failed attempt to log in

## Screenshots/Video
![Screen Shot 2022-12-01 at 8 20 15 AM](https://user-images.githubusercontent.com/83089674/205109880-6f9cf65a-75b8-42e8-a13f-d762e3ece6ff.png)

## Docker
 - Selfhosted
 - OS: Debian
 - Browser: Chrome
 - Portainer 2.15.1

## Additional logs
`10.81.234.5 - - [01/Dec/2022:16:19:18 +0000] "GET / HTTP/1.1" 200 2501 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"
10.81.234.5 - - [01/Dec/2022:16:19:18 +0000] "GET /static/css/theme.css HTTP/1.1" 304 0 "http://172.16.10.101:8888/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"
10.81.234.5 - - [01/Dec/2022:16:19:18 +0000] "GET /static/js/tabler.js HTTP/1.1" 304 0 "http://172.16.10.101:8888/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"
AXES: Repeated login failure by {username: "xxxx@xxxxxxx.com", ip_address: "10.81.234.5", user_agent: "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36", path_info: "/"}. Updated existing record in the database.
10.81.234.5 - - [01/Dec/2022:16:19:23 +0000] "POST / HTTP/1.1" 200 2701 "http://172.16.10.101:8888/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"
AXES: Repeated login failure by {username: "xxxx@xxxxxxxx.com", ip_address: "10.81.234.5", user_agent: "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36", path_info: "/"}. Updated existing record in the database.
10.81.234.5 - - [01/Dec/2022:16:19:26 +0000] "POST / HTTP/1.1" 200 2701 "http://172.16.10.101:8888/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"`

**Comment by GDay** [comment]
Thanks for the detailed report! That's a rough start.

This is very strange, because I set up a docker one yesterday with account and password and it worked fine.

Can you get into the container and then run this:

```
python manage.py shell
```
That will get you a python/django shell

Then do this:
```
from users.models import User
user = User.objects.first()
print(user.email)
```
Check what email it displays, it should be the one you put in the environment variables.

If that looks alright, then you can reset the password with:
```
user.set_password("yournewpassword")
user.save()
```

Hopefully that gets you back on track.

**Comment by jaimezapataspg** [comment]
Thanks for the quick reply; below is the output from the shell: 

`root@591db2ee44b0:/app# python manage.py shell
Python 3.9.15 (main, Nov 15 2022, 21:37:24) 
[GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> from users.models import User
>>> user = User.objects.first()
>>> print(user.email)
jdw6em@example.com
>>> 
`

That is not the email included in the env variables

**Comment by GDay** [comment]
Did you perhaps run it before you set the environment variables? That's the only reason how I could see that happening. It will create the initial account on the first run.

In any case, you can use that email address to log in. You can then create a new admin account and with that new admin account, you can delete the temporary one.

You can follow through with the steps above to change the password of that account, so you can login with it.

**Comment by rkhyd** [comment]
I also have faced the same behavior. Docker does not take into consideration the environment variables that are set and sourced.
I followed the above procedure and got a different email. This generated email was never shown in any logs as well.

I followed the above procedure of resetting the password and was able to get in.
thank you.

**Comment by GDay** [comment]
This is very strange. I am unable to reproduce this.

I just created a fresh VPS with Docker and docker-compose installed. I set up my `docker-compose.yaml` with my `ACCOUNT_EMAIL` and `ACCOUNT_PASSWORD` and ran `docker-compose up`. After a few minutes, I was able to log in with those credentials without any issues.

The only reason how I can see this issue happening is when the docker container is created before the environment variables are set.

To make this issue go away entirely, I plan on updating the creation process for the first admin account and display a form to create the first admin account (similar to how Wordpress and many other platforms do that).
